File: HTO/HTO.Web/Client/Core/Models/mobileMessage.js

(function (hto) {
	"use strict";

	function MobileMessage() {
		/// <summary>
		/// Represents a message sent from the mobile application.
		/// </summary>

		this.latitude = null;
		this.longitude = null;
		this.locationImageUrl = null;
		this.message = null;
		this.signatureImageDataUrl = null;
		this.userName = null;
	}

	hto.models.MobileMessage = MobileMessage;

}(hto));

File: HTO/HTO.Web/Server/SignalR/MobileMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HTO.Web.Server.SignalR
{
	public class MobileMessage
	{
        public decimal Latitude { get; set; }
        public decimal Longitude { get; set; }
        public string LocationImageUrl { get; set; }
		public string Message { get; set; }
        public string SignatureImageDataUrl { get; set; }
		public string UserName { get; set; }
	}
}
